components/StarSearch/StarSearchButton.tsx

```diff
@@ -1,23 +1,27 @@
 import { useLocalStorage } from "react-use";
+import clsx from "clsx";
 import Tooltip from "components/atoms/Tooltip/tooltip";
 import { StarSearchLogo } from "./StarSearchLogo";
 
 interface StarSearchButtonProps {
   onOpen: (e: React.MouseEvent<HTMLButtonElement>) => void;
+  tooltipText: string;
+  enabled: boolean;
```

Comment:
`tooltipText` is a prop now so that we can indicate whether they have access to StarSearch or not, or if they need to log in first.

`enabled` is a prop as well now, for the above reasons.